product-accelerators/apim/components/org.wso2.healthcare.apim.backendauth/src/main/java/org/wso2/healthcare/apim/backendauth/BackendAuthenticator.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.healthcare.apim.backendauth;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.synapse.MessageContext;
+import org.apache.synapse.mediators.AbstractMediator;
+import org.wso2.healthcare.apim.core.OpenHealthcareEnvironment;
+import org.wso2.healthcare.apim.core.OpenHealthcareException;
+import org.wso2.healthcare.apim.core.OpenHealthcareRuntimeException;
+import org.wso2.healthcare.apim.core.config.BackendAuthConfig;
+
+/**
+ * Backend authenticator mediator.
+ */
+public class BackendAuthenticator extends AbstractMediator {
+
+    private static final Log log = LogFactory.getLog(BackendAuthenticator.class);
+    private String authType;
```

Comment:
Can't we define the authType in the deployment toml config?

Reply:
updated in [b8da50d](https://github.com/wso2/healthcare-accelerator/pull/7/commits/b8da50dbe832d38ee8259489c5c03798ba7ca81d)